src/lib/rest/rest.cpp

```diff
@@ -518,6 +518,18 @@ static char* removeTrailingSlash(std::string path)
 */
 int servicePathSplit(ConnectionInfo* ciP)
 {
+  //
+  // Special case: empty service-path 
+  //
+  if ((ciP->httpHeaders.servicePathReceived == true) && (ciP->servicePath == ""))
```

Comment:
Might be, but that was not my intention.
I bumped into a few errors during the troubleshooting of the new implementation of orionCurl.
The changes in this PR apart from that function itself are to remedy those errors.